README.md

```diff
@@ -10,7 +29,52 @@ curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spect
 Set-ExecutionPolicy Bypass -Scope Process -Force; iex  ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/fossas/spectrometer/master/install.ps1'))
 ```
 
-## Building
+**NOTE**: In order to use the executable, you may need to add it to your PATH.
+The installer will report the installation path of the executable to make this
+easier.
+
+## Basic Usage
+
+The tool requires little-to-no configuration to run on its own.
+
+```sh
+fossa2 analyze --output # Analyze the current working directory
+# OR
+fossa2 analyze --output --basedir /path/to/project  # Analyze a specific directory.
```

Comment:
Why do we take the directory as a flag and not an argument? That UX seems unintuitive, and I'd prefer to resolve that before merging the `README`.

Reply:
I agree with this.  We can just set up an optional positional arg.  I'll PR that tomorrow, it should be quick.

Reply:
Turns out, that change isn't straightforward.  The `--basedir` option is part of the global options for the fossa executable.  I have a potential solution, but it's not great, and I don't think we should do it.  I'll link the PR here once it's uploaded.